panic in readNNormalizeRespBody

### Nuclei version: `v3.2.4`

### Current Behavior:

I've got the following panic while executing Nuclei@v3.2.4:

```
/go/pkg/mod/github.com/projectdiscovery/nuclei/v3@v3.2.4/pkg/core/executors.go:108 +0x446
created by github.com/projectdiscovery/nuclei/v3/pkg/core.(*Engine).executeTemplateWithTargets.func2 in goroutine 45698
/go/pkg/mod/github.com/projectdiscovery/nuclei/v3@v3.2.4/pkg/core/executors.go:132 +0x1d8
github.com/projectdiscovery/nuclei/v3/pkg/core.(*Engine).executeTemplateWithTargets.func2.1(0x845765?, 0x0?, 0xc002d752c0)
/go/pkg/mod/github.com/projectdiscovery/nuclei/v3@v3.2.4/pkg/tmplexec/exec.go:170 +0x332
github.com/projectdiscovery/nuclei/v3/pkg/tmplexec.(*TemplateExecuter).Execute(0xc004e9bac0, 0xc004a926e0)
/go/pkg/mod/github.com/projectdiscovery/nuclei/v3@v3.2.4/pkg/tmplexec/generic/exec.go:55 +0x2b7
github.com/projectdiscovery/nuclei/v3/pkg/tmplexec/generic.(*Generic).ExecuteWithResults(0xc005833590, 0xc004a926e0)
/go/pkg/mod/github.com/projectdiscovery/nuclei/v3@v3.2.4/pkg/protocols/http/request.go:460 +0x372
github.com/projectdiscovery/nuclei/v3/pkg/protocols/http.(*Request).ExecuteWithResults(0xc004f654a0, 0xc007016318, 0xc009c16960, 0xc009c16a50, 0xc009c16a80)
/go/pkg/mod/github.com/projectdiscovery/nuclei/v3@v3.2.4/pkg/protocols/http/request.go:396 +0x573
github.com/projectdiscovery/nuclei/v3/pkg/protocols/http.(*Request).ExecuteWithResults.func1({0xc005478480, 0x73}, 0xc00296f3e0, 0xc009c16960)
/go/pkg/mod/github.com/projectdiscovery/nuclei/v3@v3.2.4/pkg/protocols/http/request.go:746 +0x1f86
github.com/projectdiscovery/nuclei/v3/pkg/protocols/http.(*Request).executeRequest(0xc004f654a0, 0xc007016318, 0xc0011a3200, 0xc009c16a50, 0x0, 0x0?, 0x2)
/go/pkg/mod/github.com/projectdiscovery/utils@v0.0.88/http/respChain.go:146 +0xde
github.com/projectdiscovery/utils/http.(*ResponseChain).Fill(0xc00619ac60)
/go/pkg/mod/github.com/projectdiscovery/utils@v0.0.88/http/normalization.go:51 +0x142
github.com/projectdiscovery/utils/http.readNNormalizeRespBody(0xc007c4d320?, 0xc009492d80)
goroutine 45703 [running]:
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xd5cfc2]
panic: runtime error: invalid memory address or nil pointer dereference
```

I tracked down the panic to this code: https://github.com/projectdiscovery/utils/blob/v0.0.88/http/normalization.go#L42-L60

It seems that:
- line 43: `err != nil`
- line 46: `err` is reassigned to a `nil` value
- line 51: `err.Error()` on a `nil` value is called which triggers the panic

Let me know if the issue should be opened on https://github.com/projectdiscovery/utils instead.

### Expected Behavior:

It should not panic.

--- a/go.mod
+++ b/go.mod
@@ -20,7 +20,7 @@ require (
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/pkg/errors v0.9.1
 	github.com/projectdiscovery/clistats v0.0.20
-	github.com/projectdiscovery/fastdialer v0.0.67
+	github.com/projectdiscovery/fastdialer v0.0.68
 	github.com/projectdiscovery/hmap v0.0.41
 	github.com/projectdiscovery/interactsh v1.1.9
 	github.com/projectdiscovery/rawhttp v0.1.45
@@ -94,7 +94,7 @@ require (
 	github.com/projectdiscovery/tlsx v1.1.6
 	github.com/projectdiscovery/uncover v1.0.7
 	github.com/projectdiscovery/useragent v0.0.47
-	github.com/projectdiscovery/utils v0.0.89
+	github.com/projectdiscovery/utils v0.0.91
 	github.com/projectdiscovery/wappalyzergo v0.0.116
 	github.com/redis/go-redis/v9 v9.1.0
 	github.com/seh-msft/burpxml v1.0.1

